### PR TITLE
osa - output stderr journald

### PR DESCRIPTION
**Pull request by mjudeikis**
outputs all stderr to artefacts 
/cc @kargakis 

works on my box so need to get errors too :)

### CODE REVIEW

File: projects/azure/README.md [hunk 1]
```diff
@@ -152,10 +152,10 @@ docker run \
 --volume $HOME/.kube/config:/root/.kube/config \
 --volume $(pwd):/release \
 registry.svc.ci.openshift.org/ci/ci-operator:latest \
---config ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml \
+--config /release/ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml \
```
Comment (kargakis) [inline projects/azure/README.md]: All other examples assume you run from the root of the release repo? Why is this special? Please keep the PR up to the point and open a different one if we need to fix docs (although I would argue this is an unnecessary change and probably confusing because nobody is going to have `release` as a root directory.

Comment (mjudeikis) [reply projects/azure/README.md]: `--volume $(pwd):/release \` this inside of the container.

Comment (kargakis) [reply projects/azure/README.md]: ack, open a new pr to fix

Comment (mjudeikis) [reply projects/azure/README.md]: merged
